README.md

```diff
@@ -126,6 +127,49 @@ client = Redis.new(semian: {
 })
 ```
 
+#### Net::HTTP
+For the `Net::HTTP` specific Semian adapter, since many external libraries may create
+HTTP connections on the user's behalf, the parameters are instead provided
+by calling specific functions in `Semian::NetHTTP`.
+
+To give Semian parameters, assign a `proc` to `Semian::NetHTTP.raw_semian_options`
+that takes a `semian_identifier` like `http_127_0_0_1_80` or `http_github_com_80`,
+and returns a `Hash` with keys as follows, perhaps in an initialization file.
+
+```ruby
+SEMIAN_PARAMETERS = { tickets: 1,
+                      success_threshold: 1,
+                      error_threshold: 3,
+                      error_timeout: 10 }
+Semian::NetHTTP.raw_semian_options = proc do |semian_identifier|
+  # Let's make it only active for github.com
+  if semian_identifier == "http_github_com_80"
```

Comment:
I agree with the `nethttp_#{addr}_#{port}` format without any of the `tr('.', '_')` but it would be even better if the host and port was just passed separately to the block, then I wouldn't have to do any string manipulation.